packages/next-server/server/next-server.ts

```diff
@@ -78,44 +103,44 @@ export default class Server {
       })
   }
 
-  getRequestHandler () {
+  public getRequestHandler () {
```

Comment:
We don't have to write public, it's a default. 
https://github.com/este/este-typescript/blob/master/tslint.json#L11